catalog/spatial/registry/registry-rest-endpoint/pom.xml

```diff
@@ -56,6 +56,12 @@
             <artifactId>commons-lang3</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.codice.ddf.registry</groupId>
+            <artifactId>registry-ebrim-transformer</artifactId>
+            <version>${project.version}</version>
```

Comment:
I used the same foundation that was in the `FederationAdminServiceImpl` pom. Would it be better to remove it?